test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -183,6 +183,108 @@ public void AuthenticationFailure()
             }
         }
 
+#region ProvidePasswordCallback Tests
```

Comment:
Super nit: please indent the region at same level as normal code.

(hopefully it's not too annoying, just to keep consistency with the way we use regions)